components/generic/ColourSwatch.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <span
+    class="colour-palette"
+    :style="`backgroundColor: ${hexCode}`"
+    :data-qa="`colour swatch ${label}`"
+    :aria-label="`${$t('colourSwatch')} - ${label}`"
+  />
+</template>
+
+<script>
+  export default {
+    name: 'ColourSwatch',
+
+    props: {
+      label: {
```

Comment:
Replace this with a prop for the `ariaLabelledby` ID, which will need to be passed in from MoreFacetsDropdown